sqlglot/parser.py

```diff
@@ -3970,27 +3977,16 @@ def _parse_set_operations(self, this: t.Optional[exp.Expression]) -> t.Optional[
         return this
 
     def _parse_expression(self) -> t.Optional[exp.Expression]:
-        return self._parse_alias(self._parse_conjunction())
+        return self._parse_alias(self._parse_op())
 
-    def _parse_conjunction(self) -> t.Optional[exp.Expression]:
-        this = self._parse_equality()
+    def _parse_op(self) -> t.Optional[exp.Expression]:
+        return self._parse_tokens(self._parse_disjunction, self.ASSIGNMENT)
 
-        if self._match(TokenType.COLON_EQ):
-            this = self.expression(
-                exp.PropertyEQ,
-                this=this,
-                comments=self._prev_comments,
-                expression=self._parse_conjunction(),
-            )
+    def _parse_disjunction(self) -> t.Optional[exp.Expression]:
```

Comment:
i didn't really want there to be 3 methods more because of the overhead this may entil, can we just back these into 1 method?